js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -202,6 +202,10 @@ const DropDownList = DropDownEditor.inherit({
         });
     },
 
+    _isEditable: function() {
+        return this.callBase() || this.option('searchEnabled');
```

Comment:
Are u sure about this? Seems like it's wrong for lookup because of separate search field and useless for autocomplete because of no `searchEnabled` property

Reply:
>Seems like it's wrong for lookup because of separate search field

Why? At the base level this option will be used to correct cursor position and define `readonly` attribute value for default input element of widget. If it is absent (Lookup's `searchEnabled` is disabled), the attribute will not be added.

>autocomplete because of no searchEnabled property

All DropDownList-based widgets have it. It will be good if they play by the same rules.